lib/omnisharp-atom/omnisharp-atom.ts

```diff
@@ -160,6 +163,13 @@ class OmniSharpAtom {
             // This will tell us when the editor is no longer an appropriate editor
             this._activeEditor.onNext(null);
         });
+
+        atom.config.observe('omnisharp-atom.useIcons', (value) => {
```

Comment:
Could these live in `lib/omnisharp-atom/features/lib/completion-provider.ts` so they're closer to where they're used?